src/katsdpcalproc/pointing.py

```diff
@@ -97,7 +93,7 @@ def get_offset_gains(bp_gains, gains, offsets, ants, track_duration,
                 stats_N = ' '.join("%4d" % (n,) for n in abs_gain_N)
                 bp_mean = np.nanmean(np.abs(bp_gain))
                 logger.debug("%s %s %4.2f mean | %s",
-                             tuple(offset), inp, np.abs(gain), stats_mean)
+                             tuple(offset), inp, np.abs(bp_gain), stats_mean)
```

Comment:
This might be problematic, since `gain` is a scalar (and will be logged at such with the `%4.2f` format) but `bp_gain` is a vector. Maybe use `bp_mean` as well, given it's used in the next line.
```suggestion
                             tuple(offset), inp, bp_mean, stats_mean)
```